virttest/utils_misc.py

```diff
@@ -3389,33 +3389,45 @@ class struct_timespec(ctypes.Structure):
         return monotonic_time_os()
 
 
-def verify_host_dmesg(dmesg_log_file=None, ignore_result=False, level_check=3):
+def verify_dmesg(dmesg_log_file=None, ignore_result=False, level_check=3,
+                 session=None):
```

Comment:
@lmr Sure, any usage example for my reference to check in case of implementation.